examples/eks/eks.tf

```diff
@@ -38,8 +38,12 @@ resource "nirmata_cluster_type_eks" "eks-cluster-type-1" {
 resource "nirmata_cluster" "eks-cluster-1" {
   name                 = "eks-cluster-1"
   cluster_type         = nirmata_cluster_type_eks.eks-cluster-type-1.name
-   nodepools {
-      node_count                = 2
+  creation_timeout     = 30
```

Comment:
What's the unit? seconds? minutes? May be good to add to the name: `creation_timeout_seconds`.